### ISSUE TITLE
Start tagging releases -- find a better way to let people know when changes have been made

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/OTTR_Template#202]
This is related to a suggestion from Fred on Basecamp. We need a better way to let people know about new updates to the template. This also includes make the getting_started.md info more prominent (maybe into a github wiki or something?) 

At the same time we may want to think about if any reorganizing of the getting_started.md are needed.

**Comment by cansavvy** [comment]
After #197 is addressed I think I will tag a first release.

**Comment by cansavvy** [comment]
This is also related to leanbuild releases and something we should think about there as well. Like: What steps does one need to do to update their material using the latest release? And where can we publish the release notes? I think in GitHub Wiki makes sense.

**Comment by cansavvy** [comment]
Once we have releases, I'll have syncs GHA only triggered upon a new release instead of manually triggered.

**Comment by cansavvy** [comment]
Another question related to this is how should the _Leanpub repository template releases get handled? In other words should they be tied together and we make their releases identical (since they are work together anyway) or do we have it own it's own?

I think tying the releases to each other makes sense since they are already related anyway. 

Same question for Docker image and leanbuild changes. I think docker images updates are tied to the release because the dockerfile is tracked by the Bookdown repo but the leanbuild updates and release notes will be their own thing.

**Comment by cansavvy** [comment]
I've made it so when a release is created the latest docker image is built, tagged with the same release number and pushed. (I still need to test this).

**Comment by cansavvy** [comment]
This is all set. I don't want to do another test because apparently you can't fully delete the release tag. *Should* work. We'll find out when we do the release. I'll open a new issue if something needs to be fixed.
